### PR TITLE
feat(findAnswers): implement the new method

### PR DESCRIPTION
**Pull request by Haroenv**
- adds findAnswers to node / browser (not lite for now)
- fix small typo in SearchOptions
- add an integration test

### CODE REVIEW

**Review by PLNech — APPROVED** [review]

**Review by chloelbn — APPROVED** [review]

**Review by yannickcr — APPROVED** [review]

File: specs/answers.spec.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+  it("searchIndex::findAnswers", async () => {
+    const results: any = await browser.executeAsync(function(
+      credentials,
+      done
+    ) {
+      const client = algoliasearch(credentials.appId, credentials.apiKey);
+
+      // TODO: remove this customization once the engine accepts url encoded query params
+      client.transporter.userAgent.value = "answers-test";
```
Comment (eunjae-lee) [inline specs/answers.spec.ts:32]: @Haroenv to be clear, are we waiting for the engine fix, so that we can get rid of this line before merging this PR?

Comment (PLNech) [reply specs/answers.spec.ts:32]: Hey @eunjae-lee, the engine fix won't be happening soon, so I'd rather merge this and prepare a v1.1 PR for when the engine has catched up :)

Comment (eunjae-lee) [reply specs/answers.spec.ts:32]: got it. thanks for clarification!

Comment (PLNech) [reply specs/answers.spec.ts:32]: FYI, [issue opened here](https://algolia.atlassian.net/browse/SRCH-1773) :)

**Pull request merged**